examples/local/aws-operator-lab-chart/templates/configmap.yaml

```diff
@@ -10,6 +10,7 @@ data:
           id: "{{.Values.aws.accessKeyId}}"
           secret: "{{.Values.aws.secretAccessKey}}"
           session: "{{.Values.aws.sessionToken}}"
+        region: "{{.Values.aws.region}}"
```

Comment:
It would be nice to have a default value in `examples/local/aws-operator-lab-chart/values.yaml`